templates/desktop/wsl/index.html

```diff
@@ -143,6 +143,32 @@ <h3 class="p-heading--5">Mixed IT infrastructure management</h3>
       </div>
     </section>
 
+    <section class="p-section">
```

Comment:
It looks like this can be implemented using the basic section with split layout: https://vanillaframework.io/docs/patterns/basic-section#split-layout